ui/src/frontend/track_panel.ts

```diff
@@ -184,7 +184,7 @@ class TrackShell implements m.ClassComponent<TrackShellAttrs> {
         {
           class: `${highlightClass} ${dragClass} ${dropClass} ${globals.state.selectedTrackIds.has(attrs.trackState.id)? 'selected': ''}`,
           onclick: (e: MouseEvent)=>{
-            if (!e.ctrlKey) {
+            if (!(navigator.userAgent.includes('Mac')? e.metaKey : e.ctrlKey)) {
```

Comment:
Nit: this code is duplicated. I suppose we might extract it into some helper but that can wait until there's a third occurrence, which I don't expect there will be.